src/pages/public/Home/Home.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import Drawer from '../../../components/Drawer/Drawer';
+import Navbar from '../../../components/Navbar/Navbar';
+import VideoListing from './../../../components/VideoListing/VideoListing';
```

Comment:
You could add an `index.js` file in your components folder and export all the components from that and then import these altogether from there.

Reply:
True.
I try doing a barrel export and being consistent with it.
But I clearly am not consistent enough.